src/pages/careers/_model/_locations/_contact-details.js

```diff
@@ -20,6 +23,8 @@ const address_cyprus = ['Level 5, 42 Agias Filaxeos', 'Limassol 3025']
 
 const address_rwanda = ['Level 2 East Wing, Kigali Heights, KG7', 'Avenue, Kigali']
 
+const address_belarus = ['', '', 'Mink, Belarus']
```

Comment:
Why the first 2 elements are empty?

Reply:
Is it suppose to be `Minsk, Belarus`

Reply:
@AbdallaMourad typo, good catch! will fix

Reply:
Definitely, Belarus capital is Minsk.